test/app/reducers/misc.test.js

```diff
@@ -69,8 +69,8 @@ describe('misc reducers', () => {
       });
       let expectedResult = _.pickBy(devices, filterDevicesFn('win'));
       expect(actualResult).to.deep.equal(expectedResult);
-      // because nothing currently is unavailable on Windows
-      expect(_.keys(actualResult).length).to.equal(_.keys(devices).length);
+      // because we do currently have devices unavailable on Windows
```

Comment:
Can you please clarify this comment? I'm not exactly sure what it's trying to say.

Reply:
Well, to be honest, this sentence is not original work, but a simple variation of the comment from line 61 stating the same thing for Mac instead of Windows ;-)
It is supposed to explain why the next line tests for the number of available devices to be less than the the number of all devices.
Do you have any proposition how this should be better explained?
Then I could change both sentences accordingly.

Reply:
Maybe "at least one device is unavailable on Windows, so available devices should be less than all devices"?